src/object.ts

```diff
@@ -14,7 +14,7 @@ import { toBuffer, baToJSON, stripZeros } from './bytes'
  * @param data data to be validated against the definitions
  * @deprecated
  */
-export const defineProperties = function(self: any, fields: any, data: any) {
+export const defineProperties = function(self: any, fields: any, data?: any) {
```

Comment:
For reference: if I understand correctly this is a correction of the API regarding the currently existing optionality of the `data` param within the function code. It can be regarded as a bug fix, but should be pointed out in the next release notes.